DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -34,8 +34,8 @@ public void GetGroupsForCentre_returns_expected_groups()
                 AddedByLastName = "Whittaker (Developer)",
                 LinkedToField = 4,
                 LinkedToFieldName = "Job group",
-                AddNewRegistrants = true,
-                SyncFieldChanges = true
+                AddNewRegistrantsToGroup = true,
+                SyncChangesToRegistrationInfoWithGroupMembership = true

```

Comment:
So yeah, that definitely gives much better context, but I'm still not certain what they mean.
The ideal would be if each variable's name were a predicate which is true or false, rather than a command (because commands can easily be misunderstood as a concise form of a predicate; predicates generally can't be misunderstood).
So e.g. ShouldAddNewRegistrantsToGroup / NewRegistrantsHaveBeenAddedToGroup / NewRegistrantsAreInGroup are probably clearer names.

Reply:
Yeah, I'm not great at coming up with reasonably concise names for variables when they relate to a fairly complex concept. (the Sync one is definitely worse for this).

Reply:
Think you've come up with a lovely one for sync! But I reckon clarity generally trumps conciseness, especially now we have IDEs to autofill things for us.